builds/fabric8-che/assembly/assembly-wsagent-war/pom.xml

```diff
@@ -145,71 +55,4 @@
             <scope>provided</scope>
         </dependency>
     </dependencies>
-    <build>
-        <plugins>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-antrun-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>prepare-patch-files</id>
-                        <phase>generate-sources</phase>
-                    </execution>
-                    <execution>
-                        <id>copy-patched-files</id>
-                        <phase>generate-sources</phase>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.codehaus.mojo</groupId>
-                <artifactId>build-helper-maven-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>add-source-for-patched-files</id>
-                        <phase>generate-sources</phase>
-                    </execution>
-                    <execution>
-                        <id>add-resource-for-patched-files</id>
-                        <phase>generate-sources</phase>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <artifactId>maven-resources-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>copy-web-resources</id>
-                        <phase>process-sources</phase>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-patch-plugin</artifactId>
-                <executions>
-                    <execution>
-                        <id>patch-sources</id>
-                    </execution>
-                </executions>
-            </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-war-plugin</artifactId>
```

Comment:
About the war overlays, as far as I know, when you add another war as a the dependency of a war project, it will use that one to perform the overlay. So adding overlay configuration is not needed.